tests/test_operation.py

```diff
@@ -2924,28 +2925,7 @@ def test_use_legacy_opmath_fixture():
             Tensor(qml.Z(1), qml.Z(2)),
         ],
     ),
-]
-
-
-@pytest.mark.parametrize("coeffs, obs", CONVERT_HAMILTONAIN)
-def test_convert_to_hamiltonian(coeffs, obs):
-    """Test that arithmetic operators can be converted to Hamiltonian instances"""
```

Comment:
The diff here looks weird, but all that happened was the failing case in the xfail test below was moved up into the CONVERT_HAMILTONIAN list. This test is now in line 2945, and hasn't been modified.